modules/oxide/src/less/theme/components/statusbar/statusbar.less

```diff
@@ -59,7 +59,6 @@
     flex: 1 1 auto;
     height: @statusbar-text-height;
     justify-content: space-between;
-    overflow: hidden;
```

Comment:
I wonder why this was added removing it seems to still hide long help texts. @HAFRMO do you have any insights you added that line?

Reply:
@HAFRMO Can you please check and respond to Spocke's query as a Priority? TIA

Reply:
This whole ticket is why I think `box-shadow` doesn't work as an outline (and it's generally hard to make it work in a dynamic editor UI). The only way to do it is to fiddle with padding, or apply the shadow outside of the overflow, both of which make UI layout harder.
https://github.com/tinymce/tinymce/pull/9333